classes/blocks/class-hubspotform.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Hubspot Forms block class
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Blocks;
+
+/**
+ * Class HubspotForm
+ * Registers the HubspotForm block.
+ *
+ * @package P4BKS
+ * @since 0.1
+ */
+class HubspotForm extends Base_Block {
+
+	/**
+	 * Block name.
+	 *
+	 * @const string BLOCK_NAME.
+	 */
+	const BLOCK_NAME = 'hubspot-form';
+
+	/**
+	 * HubspotForm constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', [ $this, 'register_hubspotform_block' ] );
+	}
+
+	/**
+	 * Register HubspotForm block.
+	 */
+	public function register_hubspotform_block() {
+		register_block_type(
+			self::get_full_block_name(),
+			[
+				'render_callback' => [ $this, 'front_end_rendered_fallback' ],
+				'attributes' => [
+					'block_title'	=> [
```

Comment:
You can fix your code style issues automatically with phpcs/phpcbf if you have a running instance on your machine:
- Enter the php container `make php-shell`
- Get into the plugin dir `cd public/wp-content/plugins/planet4-plugin-gutenberg-blocks/`
- `./vendor/bin/phpcs` will show you the issues
- `./vendor/bin/phpcbf` will apply fixes